gqueries/output_elements/output_series/bezier_65_source_of_hot_water_in_households/solar_used_for_hot_water_in_households.gql

```diff
@@ -4,5 +4,6 @@
 - query =
     SUM(
       V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_radiation),
-      V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_thermal)      
+      V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_thermal),
```

Comment:
Same as above, this node's children don't have any solar thermal or solar radiation inputs. Am I misunderstanding what these lines do? 😆 

```ruby
CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h))
  .select { |conv| conv.input(:solar_thermal) }

# => [ <empty> ]

CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h))
  .select { |conv| conv.input(:solar_radiation) }

# => [ <empty> ]
```